### ISSUE TITLE
Improve Elastic Agent integrations logging

### ISSUE THREAD

**Issue by mbudge** [elastic/elastic-agent#380]

**Comment by jlind23** [comment]
@mbudge thanks for raising this, can I close this one as it seems to be a duplication of #100 ?

**Comment by mbudge** [comment]
Yes as long as we can monitor the state of the integrations from a single interface. 

Also collecting elastic-agent diagnostic logs is very time consuming in an enterprise environment.

**Comment by jlind23** [comment]
@nimarezainia do you know if we have any feature issue to collect agent diagnostic from fleet ui interface?

**Comment by ebeahan** [comment]
Backlog housekeeping - please reopen if we should continue to track this item.
